RIOT-OS/RIOT
# boards: Split off 128KiB version of bl*pill

### Contribution description

- Created new `bluepill-128kib` as 128KiB version of the `bluepill`
- Created new `blackpill-128kib` as 128KiB version of the `blackpill`
- Updated `openocd.cfg` to allow flashing 128KiB of ROM

### Testing procedure

Run and flash one app with ROM <= 64 KiB and one with with 64 KiB < ROM <= 128KiB.

On the `bluepill` and `blackpill` the first should flash and run, but the second should neither flash nor run. (You can build with `make CPU_MODULE=stm32f103cb BOARD=bluepill` to manually allow compiling with up to 128KiB ROM, but flashing should not work.)

On the `bluepill-128kib` and on the `blackpill-128kib`, both apps should compile and flash.

***Notes:***

1. You will need the development version of OpenOCD for the flashing >64KiB ROM to succeed with `bluepill-128kib` or `blackpill-128kib`
2. You don't need both boards, as they are basically identical (except for the LED pin and the form factor / pin mapping)

### Issues/PRs references

If merged, https://github.com/RIOT-OS/RIOT/pull/10902 is no longer needed, and thus closes #10902 ;)
